components/openstack-2023.1-jammy.yaml

```diff
@@ -28,7 +28,7 @@ images:
     ironic_db_sync: "docker.io/openstackhelm/ironic:2023.1-ubuntu_jammy"
     # these want curl which apparently is in the heat image
     ironic_manage_cleaning_network: "docker.io/openstackhelm/heat:2023.1-ubuntu_jammy"
-    ironic_retrive_cleaning_network: "docker.io/openstackhelm/heat:2023.1-ubuntu_jammy"
-    ironic_retrive_swift_config: "docker.io/openstackhelm/heat:2023.1-ubuntu_jammy"
+    ironic_retrieve_cleaning_network: "docker.io/openstackhelm/heat:2023.1-ubuntu_jammy"
+    ironic_retrieve_swift_config: "docker.io/openstackhelm/heat:2023.1-ubuntu_jammy"
```

Comment:
Similarly this is bugged upstream...


https://opendev.org/openstack/openstack-helm/src/commit/a7e8d2f7fbcdf1cb7d8c65b8e5ec144398304cb9/ironic/templates/deployment-api.yaml#L61

Reply:
fixed